python/lsst/ip/isr/overscanAmpConfig.py

```diff
@@ -36,16 +36,36 @@ class OverscanAmpConfig(pexConfig.Config):
         dtype=ParallelOverscanCorrectionTaskConfig,
         doc="Parallel overscan configuration.",
     )
+    saturation = pexConfig.Field(
```

Comment:
If I've tracked through the code correctly, `saturation` and `suspectLevel` have their secondary definition here, with the `Amplifier.getSaturation()` being the primary source.  Will those be migrated to the PTC in the future?
If `gain` is overridable here (I think these are also their secondary definition location), shouldn't `readNoise` be here too?

Reply:
I have not yet tackled the "where does saturation live?" question, and I think that should be a different ticket, so yes it can still come from the amp.
For readnoise I thought we could get it from the overscan if we don't know it, and wouldn't need to be a config.  But I can add the additional option if you think it could be useful.